views/update/update-manage-trusts-review-legal-entities.html

```diff
@@ -14,14 +14,19 @@
 
 {% set rows=[] %}
 {% for trustee in pageData.legalEntities %}
+  {% set trusteeStatus = 'Removed' if (trustee.still_involved === "No") else 'Active' %}
+  {% set trusteeRole = trusteeMacros.readableTrusteeRole(trustee.type | upper) %}
   {% set changeLink = OE_CONFIGS.UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_LEGAL_ENTITY_URL + "/" + trustee.id %}
   {% set rows = (rows.push(
     [
-      {
+      { 
```

Comment:
Trailing space after { should be deleted.

Reply:
Removed spaces